v2.0/import.md

```diff
@@ -1,43 +1,19 @@
 ---
-title: IMPORT (Experimental)
+title: IMPORT (Beta)
 summary: Import CSV data into your CockroachDB cluster.
 toc: false
 ---
 
 The `IMPORT` [statement](sql-statements.html) imports tabular data (e.g. CSVs) into a single table.
 
-{{site.data.alerts.callout_danger}}<strong>This is an experimental feature</strong>. To enable it, you must run <a href="set-cluster-setting.html"><code>SET CLUSTER SETTING experimental.importcsv.enabled = true</code></a>{{site.data.alerts.end}}
-
 {{site.data.alerts.callout_info}}For details about importing SQL dumps, see <a href="import-data.html">Import Data</a>.{{site.data.alerts.end}}
 
 <div id="toc"></div>
 
-## Glossary
-
-Term | Definition
------|-----------
-**Import file** | The tabular data file you want to import.
-**Processing node** | The single node processing the [`IMPORT`](import.html) statement/
-**Temp directory** | A location where the processing node can store data from the import file it converts to CockroachDB-compatible key-value data.<br/><br/>This directory *must* be available to all nodes using the same address (i.e., cannot use the processing node's local file storage).
-
-## Functional Overview
-
-Because importing data is a complex task, it can be useful to have a high-level understanding of the process.
-
-1. A single node receives the [`IMPORT`](import.html) request, which becomes the processing node.
-2. The processing node streams the contents of the import file, converting its contents into CockroachDB-compatible key-value data.
-3. As the key-value data is generated, the node stores it in the temp directory.
-4. Once the entire import file has been converted to key-value data, relevant nodes import key-value data from the temp directory.
-
-After the import has completed, you should also delete the files from your temp directory.
-
-## Preparation
-
 Before using [`IMPORT`](import.html), you should have:
 
 - The schema of the table you want to import.
-- The tabular data you want to import (e.g., CSV), preferably hosted on cloud storage.
-- A location to store data before it is fully imported into all your nodes (referred to in this document as a "temp" directory). This location *must* be accessible to all nodes using the same address (i.e. cannot use a node's local file storage).
+- The tabular data you want to import (e.g., CSV), preferably hosted on cloud storage. This location *must* be accessible to all nodes using the same address (i.e. cannot use a node's local file storage).
```

Comment:
nit: `i.e. cannot ...`: Let's make this a new sentence: `This means that you cannot use...`

Reply:
Done.